packages/server/tests/acceptance/ws/estimateGas.spec.ts

```diff
@@ -58,6 +58,13 @@ describe('@release @web-socket eth_estimateGas', async function () {
     gasPriceDeviation = parseFloat(expectedGas.toString() ?? '0.2');
   });
 
+  afterEach(async () => {
+    if (wsProvider) {
+      await wsProvider.destroy();
+      await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
What are we waiting for here?  Can this be replaced by some kind of check such as the wsProvider being destroyed or undefined?